libs/s25main/mapGenerator/Map.cpp

```diff
@@ -100,6 +83,15 @@ namespace rttr { namespace mapGenerator {
             resource[i] = this->resources[i];
         }
 
+        for(unsigned i = 0; i < 7; i++)
```

Comment:
Note for the future: Please no magic numbers especially in loops iterating over collections. Use `size`, range-based-for or a named constant.

Reply:
Yeah, good point. I think I left the `7` there because the old S2 didn't support more than 7 players (just an explanation, no excuse).

Reply:
Yep, I fixed that by introducing a named constant in libsiedler2, shame that there wasn't one already...